react-app/src/modules/account/account.test.js

```diff
@@ -0,0 +1,32 @@
+import * as account from './account.js';
+import naclUtils from 'tweetnacl-util';
+
+afterEach(() => account.clearAccount());
+
+test('creates keypair and checks it against server side generated keypair', async () => {
```

Comment:
Instead of "creates keypair and checks it against server side generated keypair" (which sounds like an explanation of the steps the test does) I'd write: "local and server generated key pair match".
That allows the reader to get a quicker idea of what it needs to happen instead of how.

Also, this is pretty subjective, but I usually try to read the `test('description')` as "test *that* description". It makes more sense to me.